docs/querying.md

```diff
@@ -2,10 +2,83 @@
 
 The query macro and where clause combine to give you full control over your query.
 
+## Where
+
+Where is using a QueryBuilder that allows you to chain where clauses together to build up a complete query.
+```crystal
+posts = Post.where(published: true, author_id: User.first.id)
+```
+
+It supports different operators:
+```crystal
+Post.where(:created_at, :gt, Time.now - 7.days)
+```
+
+Supported operators are :eq, :gteq, :lteq, :neq, :gt, :lt, :nlt, :ngt, :ltgt, :in, :nin, :like, :nlike
```

Comment:
@drujensen this is outside the scope of this pull, but I was just thinking. Since these are symbols, could they also be more literal operators: `:==` `:>=` etc?

Reply:
Funny you should ask.  I tried that but they are not valid symbols.  I kinda followed arel's syntax https://www.rubydoc.info/github/rails/arel/master/Arel/Predications